phidgets_api/CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ add_library(phidgets_api src/accelerometer.cpp
                          src/analog_input.cpp
                          src/analog_inputs.cpp
                          src/digital_input.cpp
+                         src/analog_output.cpp
+                         src/analog_outputs.cpp
```

Comment:
Please keep this list sorted alphabetically. The two `analog_output` lines should come before the `digital_input` and after the `analog_inputs` line.